Wave/Wave/InputTableViewCell.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  InputTableViewCell.swift
+//  Wave
+//
+//  Created by Jimmy McDermott on 7/11/16.
+//  Copyright © 2016 BlueLine Labs. All rights reserved.
+//
+
+import UIKit
+
+internal class InputTableViewCell: UITableViewCell {
+
+    lazy var textField: UITextField = {
+        let textField = UITextField(frame: CGRect(x: 16, y: 0, width: self.frame.width, height: self.frame.height))
```

Comment:
If it's offset 16px to the left, it'll go off the edge of the screen by 16px too. Width should prob be `self.frame.width - 32` (or use autolayout, cough, cough)